app/scripts/models/auth_brokers/oauth.js

```diff
@@ -179,6 +179,10 @@ define(function (require, exports, module) {
     },
 
     transformLink: function (link) {
+      if (! /^\//.test(link)) {
```

Comment:
Is it worth using a regex for this? Itooks like we're only interested in the first character, would something like the following be clearer?

```js
    if (link[0] !== '/') {
```